Tests/test_integration.py

```diff
@@ -260,9 +260,9 @@ def test_integration(client, integrations, playbook_id, options={}):
             print_error(playbook_id + ' failed on timeout')
             break
 
-        if i % DEFAULT_INTERVAL:
+        if i % DEFAULT_INTERVAL == 0:
             print 'loop no.' + str(i) + ', playbook state is ' + playbook_state
```

Comment:
Sorry about that but you should change str(i) to str(i%DEFAULT_INTERVAL) so it will be 20sec loops

Reply:
i think it should be / not %, if anything.
otherwise i will get 
loop 0
loop 0
loop 0
...

Reply:
`str(i / DEFAULT_INTERVAL + 1)`

Reply:
yep you are right